app/routes/agendapoints/edit.js

```diff
@@ -18,12 +19,17 @@ export default class AgendapointsEditRoute extends Route {
   async model() {
     const { documentContainer, returnToMeeting } =
       this.modelFor('agendapoints');
-    const standardTemplates = this.standardTemplate.fetchTemplates.perform();
+    const standardTemplates =
+      await this.standardTemplate.fetchTemplates.perform();
```

Comment:
`await`s here will mean the requests run in series, I think we can remove the awaits and just pass the promises to `RSVP` so they run in parallel.

Reply:
Done :)